apache/zeppelin
# [ZEPPELIN-3540] Show Spark inline UI in Non-Secure mode

### What is this PR for?
After running a paragraph via Spark interpreter, we will be showing a "Spark UI" link. On clicking it, we show the Spark UI with the respective running job running.

### What type of PR is it?
[Improvement]

### What is the Jira issue?
[[ZEPPELIN-3540](https://issues.apache.org/jira/browse/ZEPPELIN-3540)]

### How should this be tested?
Using Spark interpreter, run a paragraph. On successful execution, we will see a Spark UI link(as shown in screenshot below) Click on the link and the pop up modal opens up showing the Spark job in the Zeppelin context.

### Screenshots (if appropriate)
![spark-ui-link](https://user-images.githubusercontent.com/6761317/41392631-778ed0e2-6fbf-11e8-94c4-cfab750184cf.png)

![webui-modal](https://user-images.githubusercontent.com/6761317/41392636-80a4d000-6fbf-11e8-8717-c24d70f1ebcf.png)
